README.md

```diff
@@ -129,10 +129,14 @@ window.plugins.speechRecognition.requestPermission(
 This method requests access permission to system resources if it was not granted before.
 
 
-### Ionic 2 Usage
+### Ionic 3 Usage
```

Comment:
get rid of the "3" here - this code will work in all Ionic versions, also 2 - it is actually "Ionic Native 3" that brought these changes.